README.md

```diff
@@ -1,2 +1,24 @@
 # ssc-js-sandbox
-Fortify Software Security JavaScript sandbox
+
+A sandbox project including samples and workflows with the SSC REST API has been released. This project is intended as a tutorial to encourage learning the API and a quick way to get started.
+> This API is always tested with the latest GA release of SSC, starting with 17.2
+
+[Documentaion](https://github.com/fortify/ssc-js-sandbox-docs)
```

Comment:
This link currently does not contain any documentation. 
It just says this:

ssc-js-sandbox-docs
Documentation for the ssc-js-sandbox project

Is it OK to publish link util all necessary information is added there?